session_items.py

```diff
@@ -1,10 +1,6 @@
 from flask import session
```

Comment:
Empty file above can be removed

Reply:
if I remove the 'from flask import session' it returns with errors??